### ISSUE TITLE
Add Option for Data Number Markings on Spatial Layers (without Contours)

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#473]
Give the user the ability to overlay data labels to spatial layers. This is just like adding contour lines without the lines. So allows the user to have an idea on what the magnitude is just by looking at the map and not opening the legend. Like soundings on a maritime chart.

Similar to a temperature plot that weather news stations show

![image](https://user-images.githubusercontent.com/82955395/170545278-89ecd36c-f26a-424e-8fcf-224fd87ba178.png)

**Comment by ndellicarpini** [comment]
this would have to be done w/ pyxms or zms right? Otherwise we would need to request data at basically every point on the map in order to find where good places for the numbers to be shown.

**Comment by mpiannucci** [comment]
Yup this is a potential pyxms feature that was already started by a former employee so I threw it up in case we got time to do it

**Comment by mpiannucci** [comment]
See https://github.com/asascience/pyXMS/pull/67 for WIP from before I started here haha

**Comment by ndellicarpini** [comment]
Do we want to put this level of effort into pyxms? This and https://github.com/asascience/ott-coops/issues/222 would be good improvements to pyxms, but would be very time consuming.

**Comment by mpiannucci** [comment]
Lets hold off for the moment
